src/main/java/domain/board/BoardGenerator.java

```diff
@@ -0,0 +1,156 @@
+        board.put(nodeByPoint.get(Point.of(4, 7)), new Byeong(Team.HAN));
+        board.put(nodeByPoint.get(Point.of(4, 9)), new Byeong(Team.HAN));
+
+        board.put(nodeByPoint.get(Point.of(3, 2)), new Po(Team.HAN));
+        board.put(nodeByPoint.get(Point.of(3, 8)), new Po(Team.HAN));
+
+        board.put(nodeByPoint.get(Point.of(2, 5)), new Wang(Team.HAN));
+
+        board.put(nodeByPoint.get(Point.of(1, 1)), new Cha(Team.HAN));
+        board.put(nodeByPoint.get(Point.of(1, 4)), new Sa(Team.HAN));
+        board.put(nodeByPoint.get(Point.of(1, 6)), new Sa(Team.HAN));
+        board.put(nodeByPoint.get(Point.of(1, 9)), new Cha(Team.HAN));
+        Deque<Piece> sangMaOrder = createSangMaOrder(sangMaOrderCommand, Team.HAN);
+        for (Point point : sangMaPoints) {
+            board.put(nodeByPoint.get(point), sangMaOrder.removeFirst());
+        }
+    }
+
+    private void initializeChoPiecePosition(List<Point> sangMaPoints,
+                                            SangMaOrderCommand sangMaOrderCommand,
+                                            Map<Point, Node> nodeByPoint, Map<Node, Piece> board) {
+        board.put(nodeByPoint.get(Point.of(7, 1)), new Byeong(Team.CHO));
+        board.put(nodeByPoint.get(Point.of(7, 3)), new Byeong(Team.CHO));
+        board.put(nodeByPoint.get(Point.of(7, 5)), new Byeong(Team.CHO));
+        board.put(nodeByPoint.get(Point.of(7, 7)), new Byeong(Team.CHO));
+        board.put(nodeByPoint.get(Point.of(7, 9)), new Byeong(Team.CHO));
+
+        board.put(nodeByPoint.get(Point.of(8, 2)), new Po(Team.CHO));
+        board.put(nodeByPoint.get(Point.of(8, 8)), new Po(Team.CHO));
+
+        board.put(nodeByPoint.get(Point.of(9, 5)), new Wang(Team.CHO));
+
+        board.put(nodeByPoint.get(Point.of(10, 1)), new Cha(Team.CHO));
+        board.put(nodeByPoint.get(Point.of(10, 4)), new Sa(Team.CHO));
+        board.put(nodeByPoint.get(Point.of(10, 6)), new Sa(Team.CHO));
+        board.put(nodeByPoint.get(Point.of(10, 9)), new Cha(Team.CHO));
+        Deque<Piece> sangMaOrder = createSangMaOrder(sangMaOrderCommand, Team.CHO);
+        for (Point point : sangMaPoints) {
+            board.put(nodeByPoint.get(point), sangMaOrder.removeFirst());
+        }
+    }
+
+    private Deque<Piece> createSangMaOrder(SangMaOrderCommand sangMaOrderCommand, Team team) {
+        List<PieceType> pieceTypes = sangMaOrderCommand.getPieceTypes();
+        Deque<Piece> pieces = new ArrayDeque<>();
+        for (PieceType pieceType : pieceTypes) {
+            pieces.addLast(createPiece(pieceType, team));
+        }
+        return pieces;
+    }
+
+    private Piece createPiece(PieceType pieceType, Team team) {
+        Piece piece;
+        switch (pieceType) {
+            case SANG -> piece = new Sang(team);
+            case MA -> piece = new Ma(team);
+            default -> throw new IllegalArgumentException("[ERROR] 상 또는 마가 아닙니다.");
+        }
+        return piece;
+    }
```

Comment:
java 13부터는 [switch expression](https://www.baeldung.com/java-switch#switch-expressions)을 제공하는데요. 
이 방식을 활용하면 아래와 같이 변수를 일단 선언한 뒤 할당하는 방식을 사용할 필요 없이 바로 결과를 변수로 선언과 동시에 할당 혹은 return 할 수 있을 것 같아요. 😃 

```suggestion
    private Piece createPiece(PieceType pieceType, Team team) {
        return switch (pieceType) {
            case SANG -> new Sang(team);
            case MA -> new Ma(team);
            default -> throw new IllegalArgumentException("[ERROR] 상 또는 마가 아닙니다.");
        };
    }
```

Reply:
감사합니다 적용했습니다! 😁